src/pudl/transform/eia861.py

```diff
@@ -1572,7 +1579,7 @@ def operational_data(tfr_dfs):
     ###########################################################################
 
     transformed_od = (
```

Comment:
I noticed here that there are some column headers that could potentially but not necessarily be normalized. They all start with 'revenue' (below). Do you think I aught to normalize them too? 

`revenue_from_delivery_customers`
`revenue_from_sales_for_resale`
`revenue_from_credits_or_adjustments`
`revenue_from_transmission`
`revenue_from_other`
`revenue_total`

Reply:
it looks like if we were to normalize these columns they'd need to be pulled out into a separate table. because the new idx_cols would be different for this set of columns than it would be for the rest. Is that true?

I suppose it would be pretty easy to grab just these revenue columns, use `_tidy_class_dfs` and save two tables to `tfr_dfs`. And that way you'd only have to turn 1000s of dollars back into dollars on one column.

That seems like it might be nice, but you are more familiar with this table than I am.

Reply:
This is an interesting point because I think there are some other tables where the normalization only applies to some of the columns. For instance, the net_metering and non_net_metering tables use of fuel_class as a normalizer does not apply to every column. Is this something that should be corrected? Happy to hop on the phone to explain.